backend/app.js

```diff
@@ -0,0 +1,44 @@
+const express = require('express');
+const cors = require('cors');
+const helmet = require('helmet');
+const responseTime = require('response-time');
+const routes = require('./app/routes');
+const { errorHandler } = require('./helpers/error');
+require('colors');
+require('./helpers/dbConnection');
+
```

Comment:
Why do you want to remove this file to app.js?

Reply:
This with all express configuration and exporting app is usually named as app.js. That's why I changed it to app.js.

This format is even followed in express [generator](https://expressjs.com/en/starter/generator.html)( famous boiler plate for express servers ).

Reply:
Yes, even if we change the file name to app.js then index.js shouldn't be there and the cluster code should be ideally refactored to another helper function maybe cluster.js

Reply:
Why do we still have 2 files in the root directory? Ideally there should be just one file either `index.js` or `app.js`

Reply:
App.js  has all the express configuration and exports app. 

While the index.js launches and http server with / without clustering. 

This will help in future while deploying in different environments like index.js for normal server,  lambda.js for deploying in lambda, etc each one will import app and launch it in a desired way. 

If you want I can merge them but you will surely need this architecture in future. 

BTW this structure is also followed by express Generator (details mentioned in prevent comments).